lib/content.js

```diff
@@ -1,4 +1,5 @@
 var lodash = require('lodash');
+var url = require('./url-parser');
```

Comment:
Just to be clear that this isn't a plain URL object, just call it something like `urlParser`, which is a bit closer to the module's name :+1: